wcfsetup/install/files/lib/system/listView/user/TeamListView.class.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace wcf\system\listView\user;
+
+use wcf\data\user\UserProfileList;
+
+/**
+ * List view for the team list.
+ *
+ * @author      Marcel Werk
+ * @copyright   2001-2025 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since       6.2
+ */
+class TeamListView extends MembersListView
+{
+    public function __construct(private readonly int $groupID)
```

Comment:
I think it would be better to expect a `UserGroup` object instead. This is already available in `TeamPage` and serves as a validation that the ID is most likely correct.